ci/scripts/target-location.sh

```diff
@@ -30,8 +30,8 @@ then
   REDIR='/dev/null'
 fi
 
-REL_PATH=$(${REPO_TOP}/bazelisk.sh outquery "$@" 2>$REDIR)
+REL_PATH=$(${REPO_TOP}/bazelisk.sh cquery --output=files "$@" 2>$REDIR)
 readonly REL_PATH
 REPO_EXECROOT=$(${REPO_TOP}/bazelisk.sh info --show_make_env execution_root)
 readonly REPO_EXECROOT
-echo "${REPO_EXECROOT}/${REL_PATH}"
+echo "${REL_PATH}" | awk -v root="$REPO_EXECROOT" '{print root "/" $0}'
```

Comment:
Do you remember what this change was for? It looks equivalent to me

Reply:
We discussed offline, `awk` adds the root per line where the original prefixed the whole thing